praw/reddit.py

```diff
@@ -273,8 +276,11 @@ def _prepare_objector(self):
             self._objector.register(kind, klass)
 
     def _prepare_prawcore(self):
-        requestor = Requestor(USER_AGENT_FORMAT.format(self.config.user_agent),
-                              self.config.oauth_url, self.config.reddit_url)
+        requestor = self.requestor
```

Comment:
Could we use the `or` syntax like you did with prawcore? Assuming the parameter name is `custom_requestor`

    requestor = custom_requestor or Requestor(
        USER_AGENT_FORMAT...)